src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -393,7 +393,10 @@ def get_deployment_layout(self) -> List["OSD"]:
         functions.
         '''
         devices = self.args.devices
-        fast_devices = self.args.db_devices
+        if self.args.block_db_size is not None:
```

Comment:
is this `block_db_size` something we're going to need cephadm to pass in in order to have the orchestrator make use of this?

Reply:
This parameter already exists. This new feature doesn't require any changes on the cephadm side. See the commit message for examples.